src/zenml/analytics/context.py

```diff
@@ -77,7 +77,12 @@ def __enter__(self) -> "AnalyticsContext":
         from zenml.config.global_config import GlobalConfiguration
 
         gc = GlobalConfiguration()
-        self.analytics_opt_in = gc.analytics_opt_in
+        store_info = gc.zen_store.get_store_info()
```

Comment:
Maybe push this below the `if self.in_server`? It is not needed anywhere else.

Reply:
It is needed below.

Reply:
Yes, found it now, thanks